walkthroughs/howto-http-retries/README.md

```diff
@@ -0,0 +1,72 @@
+# HTTP Retry Policy Example
+
+This example shows how we can set retry duration and attempts within route configurations. In this mesh we have a frontend service and a color service. The frontend service is responsible for forwarding a header called _statuscode-header_ to the 
+color service. If this header is present the color service is expecting it to be set to either 200, 500, or 503. If it is not set then the color service defaults to 200.
+The envoy sidecar in frontend will know to retry based on the configured retry policy when it is applicable, in this case when a 503 is sent through the header _statuscode-header_.  
+
+## Setup
+
+1. Clone this repository and navigate to the walkthrough/http-retry-policy folder, all commands will be ran from this location
+
+2. Your account id:
```

Comment:
nit: pro-tip, if you just do 1. repeatedly, markdown should be associating it to the same list and auto-numbering.